### DISCUSSION TITLE
How can I start app in dev both on my pc and on phone that is connected to it?

### DISCUSSION THREAD

**Discussion by StepanZagray** [tauri-apps/tauri#12040] [Q&A]
How can I start app in dev both on my pc and on phone that is connected to it?
By that I mean two dev instances, one compiles for android, one compiles for windows.
I'm also interested is it possible to have two dev instances on one machine.(Tauri starts two seperate dev apps on one pc)

**Comment by FabianLars** [comment]
> How can I start app in dev both on my pc and on phone that is connected to it?
> By that I mean two dev instances, one compiles for android, one compiles for windows.

You can just open 2 terminals, one for `tauri android` dev and one for `tauri dev`, that _should_ work. If your frontend dev server is configured to crash if a port is already taken you either need to disable that, remove the beforeDevCommand (via the -c flag for example) for one or both commands (in case of the latter you'll need a third terminal for the frontend dev server), or start the desktop dev via `cargo run` instead of `tauri dev`.

>  I'm also interested is it possible to have two dev instances on one machine.(Tauri starts two seperate dev apps on one pc)

Sure, just make sure that each project uses its own port.
